stylesheets/design-patterns/_media-player.scss

```diff
@@ -222,7 +227,9 @@
         height: 100%;
         padding-bottom: 1px;
         width: 20px;
-        color: rgba(0,0,0,0.5);
+        // Allow RGBA here, it is the only colour value with alpha transparency
+        // scss-lint:disable ColorVariable
+        color: rgba(0, 0, 0, 0.5);
```

Comment:
I had a similar linting issue on [static](https://github.com/alphagov/static/pull/745/files#diff-17beb2dcf03801a29d1ac8f73879ca20R434) and used SCSS's `transparentize` with `$black` to meet linting without needing to disable a rule.